infra/k8s/greptime/eks-deployment-runbook.md

```
@@ -70,7 +70,7 @@ eksctl create podidentityassociation -f infra/k8s/greptime/pod-identity-associat
 Restart Greptime workloads so pods refresh credentials after the secret and Pod Identity association are attached:
 
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)